js/views/messageattachment.js

```diff
@@ -68,14 +68,14 @@ define(function(require) {
 			var folder = require('state').currentFolder;
 			var messageId = this.model.get('messageId');
 			var attachmentId = this.model.get('id');
-			// Loading feedback
-			this.getUI('saveToCloudButton').removeClass('icon-folder')
```

Comment:
Why did you remove this? Doesn't this remove the initial loading feedback?

Reply:
I wanted to make the button show loading only when user clicks Choose to choose a directory. Shouldn't the button stay the same unless user confirms saving?

Reply:
@haivp3010 the loading feedback should appear as soon as the button is clicked already. So if the popover does _not_ overlap the button you see the loading feedback already.

Reply:
@jancborchardt I think that the purpose of the loader is to represent file saving status. For the file picker, it does have its own loading spinner while loading up, so why should we leave the former as loading while it's function is actually not being executed?

Reply:
A similar situation can be seen here https://developers.google.com/drive/v3/web/savetodrive, where the Save to drive button does not show as loading while still choosing a folder.
![image](https://user-images.githubusercontent.com/7151454/31245868-da43fc8a-aa35-11e7-892f-8de6cfa73857.png)
The loader only shows when the file is actually being uploaded.
![image](https://user-images.githubusercontent.com/7151454/31245966-2fffbd62-aa36-11e7-94f4-20f1a1b8f075.png)

Of course, different systems have different ways to do things, but I think it's just not common practice to show the loading feedback so early while still choosing a directory.